guardian/support-frontend
# Imagery changes to Guardian Weekly Page

## What are you doing in this PR?
Updating the imagery for Guardian Weekly on support frontend 

[**Trello Card**](https://trello.com/c/Sd0dd8lD/6-support-frontend-imagery-changes-to-guardian-weekly-page)

## Why are you doing this?
To reflect the new look and up-to-date journalism which may encourage more subscriptions to the magazine

## How to test

On support frontend, check that the images for Guardian Weekly are as expected (see trello card for image links). Change screen size to check that images are correctly displayed.

## How can we measure success?



## Have we considered potential risks?



## Accessibility test checklist


## Screenshots
`[environment variable]/uk/subscribe`
**DESKTOP**
before
![image](https://github.com/guardian/support-frontend/assets/115992455/83eba5f9-3961-4031-912a-af95ae9404a8)

after
![image](https://github.com/guardian/support-frontend/assets/115992455/a7e69280-4d7a-4159-b6b0-d49035a410b6)

**TABLET (1024 x 1366)**
before
<img width="508" alt="image" src="https://github.com/guardian/support-frontend/assets/115992455/2778c6c7-8542-4d45-be17-fef5a6d4ca1b">

after
![image](https://github.com/guardian/support-frontend/assets/115992455/8c143aec-8c75-4efc-b1fd-48380128b7a1)

**MOBILE (360 x 740)**
before
<img width="361" alt="image" src="https://github.com/guardian/support-frontend/assets/115992455/98def417-0489-4e63-a3b0-1d15811fa509">

after
![image](https://github.com/guardian/support-frontend/assets/115992455/a5caf9bb-bf48-424f-a977-14315873b6a2)

`[environment variable]/uk/subscribe/weekly`
**DESKTOP**
before
![image](https://github.com/guardian/support-frontend/assets/115992455/46a83c10-8ca9-488c-868a-5b29897fcffb)

after
![image](https://github.com/guardian/support-frontend/assets/115992455/0d52e62c-9bb2-4d1e-a926-45535fcc9409)

**TABLET (1024 x 1366)**
before
<img width="496" alt="image" src="https://github.com/guardian/support-frontend/assets/115992455/06353de1-26c5-416d-8c57-607b8b6ede8a">

after
![image](https://github.com/guardian/support-frontend/assets/115992455/f67f6fe3-1f08-400e-adbe-04581f38f3b9)

**MOBILE (360 x 740)**
before
<img width="357" alt="image" src="https://github.com/guardian/support-frontend/assets/115992455/cf734d71-24bb-4c28-9bbf-789ec03e3ead">
<img width="358" alt="image" src="https://github.com/guardian/support-frontend/assets/115992455/15495008-523f-4a5e-add0-3b62f4f3ae81">

after
![image](https://github.com/guardian/support-frontend/assets/115992455/3ead60dc-16f7-401c-b5b5-64563101f643)
![image](https://github.com/guardian/support-frontend/assets/115992455/4698a6ab-367e-42cb-91db-b1fddbd5e527)

`[environment variable]/subscribe/weekly/checkout`
**DESKTOP**
before
![image](https://github.com/guardian/support-frontend/assets/115992455/d26fe907-096c-489e-912e-1ca6466b03b5)

after
![image](https://github.com/guardian/support-frontend/assets/115992455/ccd7646c-b0de-44dc-9f15-f0baedaf747d)

**TABLET (1024 x 1366)**
before
<img width="507" alt="image" src="https://github.com/guardian/support-frontend/assets/115992455/26f0bfe8-e33f-4834-803b-a7a8e772d16b">

after
![image](https://github.com/guardian/support-frontend/assets/115992455/77b0ed17-cd44-4e77-af28-983e54dbca29)

**MOBILE** no changes as image not shown
